addons/l10n_kz/demo/demo_company.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="partner_demo_company_kz" model="res.partner">
+        <field name="name">KZ Company</field>
+        <field name="vat">090241007868</field>
+        <field name="street">Respublika Avenue 4</field>
+        <field name="city">Shymkent</field>
+        <field name="country_id" ref="base.kz"/>
+        
+        <field name="zip">160012</field>
+        <field name="phone">+77252535000</field>
+        <field name="email">info@meyramgroup.kz</field>
+        <field name="website">https://meyramgroup.kz</field>
+    </record>
+
+    <record id="demo_company_kz" model="res.company">
+        <field name="name">KZ Company</field>
+        <field name="partner_id" ref="partner_demo_company_kz"/>
+    </record>
+
+    <function model="res.company" name="_onchange_country_id">
+        <value eval="[ref('demo_company_kz')]"/>
```

Comment:
what are you doing?

Reply:
I guess we should cover this questions in docs https://github.com/odoo/documentation/pull/1334 :wink:

Reply:
Might be overkill IMO, but why not...

Reply:
Because even Quentin didn't know what is going on here :D